internal/compose/compose.go

```diff
@@ -23,6 +23,12 @@ import (
 	"github.com/elastic/elastic-package/internal/signal"
 )
 
+// waitForHealthyTimeout is the maximum duration for WaitForHealthy().
+var waitForHealthyTimeout = time.Duration(10 * time.Minute)
```

Comment:
Nit. No need to convert to `time.Duration`, a number multiplying a duration is already a duration.